app/models/assignment.rb

```diff
@@ -1,4 +1,9 @@
 class Assignment < ApplicationRecord
   belongs_to :client
   belongs_to :volunteer
+  belongs_to :user
```

Comment:
The user_id of the assignment and the user_id of the client/volunteer might not be the same. user_id of the assignment is the user who created the assignment. Client user is the user who registered the client and Volunteer user is, if I remember correctly, the user who the volunteer becomes when she gets an account. So there is high chance they are not the same person.